Tests/FoundationEssentialsTests/StringTests.swift

```diff
@@ -22,44 +22,117 @@ import TestSupport
 
 final class StringTests : XCTestCase {
     // MARK: - Case mapping
-
+    
```

Comment:
And to round out the formatting nits, somehow this added empty whitespace lines.